api/src/main/java/org/openmrs/module/appointments/scheduler/tasks/ReminderForAppointment.java

```diff
@@ -24,7 +31,16 @@ public void execute() {
         String schedulerReminderTime = administrationService.getGlobalPropertyObject("SchedulerReminderBeforeHours").getPropertyValue();
         List<Appointment> appointments = appointmentsService.getAllAppointmentsReminder(schedulerReminderTime);
         for (Appointment appointment: appointments) {
-           appointmentsService.sendAppointmentReminderSMS(appointment);
+            PersonAttribute phoneNumber = appointment.getPatient().getAttribute("phoneNumber");
```

Comment:
this assumes that the phone number attribute is phoneNumber. which for now is ok, I guess